app/uk/gov/hmrc/agentuserclientdetails/config/AppConfig.scala

```diff
@@ -18,9 +18,32 @@ package uk.gov.hmrc.agentuserclientdetails.config
 
 import javax.inject.{Inject, Singleton}
 import play.api.Configuration
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 @Singleton
-class AppConfig @Inject()(config: Configuration) {
+class AppConfig @Inject()(servicesConfig: ServicesConfig) {
 
-  val appName: String = config.get[String]("appName")
+  private def getConf(key: String) =
+    servicesConfig.getConfString(key, throw new RuntimeException(s"config $key not found"))
+  private def baseUrl(key: String) = servicesConfig.baseUrl(key)
+
+  val appName: String = "agent-user-client-details"
+
+  val citizenDetailsBaseUrl: String = baseUrl("citizen-details")
+
+  val enrolmentStoreProxyUrl = servicesConfig.baseUrl("enrolment-store-proxy")
+
+  val desBaseUrl: String = baseUrl("des")
+  val desEnvironment: String = getConf("des.environment")
+  val desAuthToken: String = getConf("des.authorization-token")
+
+  val ifPlatformBaseUrl: String = baseUrl("if")
+  val ifEnvironment: String = getConf("if.environment")
+  val ifAuthTokenAPI1712: String = getConf("if.authorization-token.API1712")
+  val ifAuthTokenAPI1495: String = getConf("if.authorization-token.API1495")
+
+  val desIFEnabled: Boolean = servicesConfig.getBoolean("des-if.enabled")
```

Comment:
we don't need this...IF is permanently enabled so use IF for trusts and PPT, all others use DES (I think!)

Reply:
Cool, I'll get rid of it